--- src/chat_control.py	2016-05-26 17:49:16.811472595 +0300
+++ src/chat_control.py	2016-05-26 17:50:19.481473430 +0300
@@ -656,7 +656,10 @@ class ChatControlBase(MessageControl, Ch
     def _conv_textview_key_press_event(self, widget, event):
         # translate any layout to latin_layout
         keymap = gtk.gdk.keymap_get_default()
-        keycode = keymap.get_entries_for_keyval(event.keyval)[0][0]
+        keyval = keymap.get_entries_for_keyval(event.keyval)
+        if not keyval or not keyval[0]:
+            return False
+        keycode = keyval[0][0]
         if (event.state & gtk.gdk.CONTROL_MASK and keycode in (self.keycode_c,
         self.keycode_ins)) or (event.state & gtk.gdk.SHIFT_MASK and \
         event.keyval in (gtk.keysyms.Page_Down, gtk.keysyms.Page_Up)):
