--- Telegram/Telegram.pro	2016-06-21 19:03:27.390433026 +0300
+++ Telegram/Telegram.pro	2016-06-21 19:05:39.290434783 +0300
@@ -10,7 +10,6 @@ CONFIG(debug, debug|release) {
     DESTDIR = ./../Debug
 }
 CONFIG(release, debug|release) {
-    DEFINES += CUSTOM_API_ID
     OBJECTS_DIR = ./../ReleaseIntermediate
     MOC_DIR = ./GeneratedFiles/Release
     RCC_DIR = ./GeneratedFiles
@@ -385,10 +384,6 @@ HEADERS += \
   ./SourceFiles/pspecific_mac.h
 }
 
-SOURCES += \
-  ./ThirdParty/minizip/zip.c \
-  ./ThirdParty/minizip/ioapi.c
-
 CONFIG += precompile_header
 
 PRECOMPILED_HEADER = ./SourceFiles/stdafx.h
@@ -418,16 +413,16 @@ CONFIG(debug, debug|release) {
 include(qt_static.pri)
 
 INCLUDEPATH += \
-               /usr/local/include\
-               /usr/local/include/opus\
                ./SourceFiles\
                ./GeneratedFiles\
-               ./ThirdParty/minizip\
-               ./../../Libraries/breakpad/src
+               /usr/include/opus\
+               /usr/include/minizip\
+               /usr/include/breakpad
 
-INCLUDEPATH += "/usr/include/libappindicator-0.1"
-INCLUDEPATH += "/usr/include/gtk-2.0"
+INCLUDEPATH += "/usr/include/libappindicator3-0.1"
+INCLUDEPATH += "/usr/include/gtk-3.0"
 INCLUDEPATH += "/usr/include/glib-2.0"
+INCLUDEPATH += "/usr/lib/glib-2.0/include"
 INCLUDEPATH += "/usr/lib/x86_64-linux-gnu/glib-2.0/include"
 INCLUDEPATH += "/usr/lib/i386-linux-gnu/glib-2.0/include"
 INCLUDEPATH += "/usr/include/cairo"
@@ -444,9 +439,7 @@ LIBS += -ldl -llzma -lopenal -lavformat
 LIBS += $${QT_TDESKTOP_PATH}/plugins/platforminputcontexts/libcomposeplatforminputcontextplugin.a \
         $${QT_TDESKTOP_PATH}/plugins/platforminputcontexts/libibusplatforminputcontextplugin.a \
         $${QT_TDESKTOP_PATH}/plugins/platforminputcontexts/libfcitxplatforminputcontextplugin.a
-LIBS += /usr/local/lib/libz.a
-LIBS += /usr/local/lib/libxkbcommon.a
-LIBS += ./../../../Libraries/breakpad/src/client/linux/libbreakpad_client.a
+LIBS += -lminizip -lz -lxkbcommon -lcrypto -lssl -lbreakpad_client
 
 RESOURCES += \
     ./Resources/telegram.qrc \
