diff --git a/Telegram/Telegram.pro b/Telegram/Telegram.pro
index 9f26dc0..4a4fd12 100644
--- a/Telegram/Telegram.pro
+++ b/Telegram/Telegram.pro
@@ -10,7 +10,6 @@ CONFIG(debug, debug|release) {
 	DESTDIR = ./../Debug
 }
 CONFIG(release, debug|release) {
-	DEFINES += CUSTOM_API_ID
 	OBJECTS_DIR = ./../ReleaseIntermediate
 	MOC_DIR = ./GeneratedFiles/Release
 	RCC_DIR = ./GeneratedFiles
@@ -489,10 +488,6 @@ HEADERS += \
 	./SourceFiles/platform/mac/main_window_mac.h
 }
 
-SOURCES += \
-	./ThirdParty/minizip/zip.c \
-	./ThirdParty/minizip/ioapi.c
-
 CONFIG += precompile_header
 
 PRECOMPILED_HEADER = ./SourceFiles/stdafx.h
@@ -523,9 +518,7 @@ include(qt_static.pri)
 
 INCLUDEPATH += \
 	./SourceFiles\
-	./GeneratedFiles\
-	./ThirdParty/minizip\
-	./../../Libraries/breakpad/src
+	./GeneratedFiles
 
 CONFIG += link_pkgconfig
 PKG_CONFIG = $$pkgConfigExecutable()
@@ -545,7 +538,7 @@ PKGCONFIG += \
 # PKGCONFIG links it like "-L/usr/local/lib -lxkbcommon"
 # which makes a dynamic link which leads to segfault in
 # QApplication() -> createPlatformIntegration -> QXcbIntegrationPlugin::create
-	#xkbcommon\
+	xkbcommon\
 	openal\
 	libavformat\
 	libavcodec\
@@ -556,6 +549,7 @@ PKGCONFIG += \
 	libva\
 	libssl\
 	libcrypto\
+	minizip\
 	zlib\
 	liblzma
 
@@ -565,14 +559,6 @@ LIBS += $${QT_TDESKTOP_PATH}/plugins/platforminputcontexts/libcomposeplatforminp
         $${QT_TDESKTOP_PATH}/plugins/platforminputcontexts/libibusplatforminputcontextplugin.a \
         $${QT_TDESKTOP_PATH}/plugins/platforminputcontexts/libfcitxplatforminputcontextplugin.a
 
-LIBS += ./../../../Libraries/breakpad/src/client/linux/libbreakpad_client.a
-
-# In order to work libxkbcommon must be linked statically,
-# PKGCONFIG links it like "-L/usr/local/lib -lxkbcommon"
-# which makes a dynamic link which leads to segfault in
-# QApplication() -> createPlatformIntegration -> QXcbIntegrationPlugin::create
-LIBS += /usr/local/lib/libxkbcommon.a
-
 RESOURCES += \
 	./Resources/telegram.qrc \
 	./Resources/telegram_linux.qrc \
